parquet-hadoop/src/test/java/org/apache/parquet/hadoop/TestParquetFileWriter.java

```diff
@@ -89,10 +89,13 @@
 
 import org.apache.parquet.hadoop.example.GroupWriteSupport;
 import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
 import org.mockito.Mockito;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+@RunWith(Parameterized.class)
 public class TestParquetFileWriter {
```

Comment:
1. because it does a read afterwards.
2. and when getting the pr to pass all the tests with vector IO turned on for all of them, this was one which found a bug. Parameterization ensures it stays fixed and with better coverage.

Reply:
It make sense, thanks. However, it is a bit misleading to me to set the vector IO related config for the writer as well. What do you think about separate the `Configuration` instances for the writer and the reader and only enable vector IO for the reader?

Reply:
started this, but it doesn't work. because filesystems are cached on path; once a writer has retrieved the fs instance for a path, the reader gets it back too. sorry

Reply:
summary. cantfix